index.html

```diff
@@ -5,7 +5,7 @@
     <link rel="stylesheet" href="css/style.css">
     <link href='https://fonts.googleapis.com/css?family=Poppins:400,500,700,900' rel='stylesheet'>
   </head>
-  <body>
+  <body onload="initialExec();">
```

Comment:
- [ ] As I can see in the HTML you kept your HTML for the work section the same but acoording to the requirements it is required to add dynamically like the pop up so please remove all the hardcoded from the HTML and generate it using JS

![image](https://user-images.githubusercontent.com/58553711/197085712-72f8d632-081e-4b0b-a66c-03f6e6f31f6d.png)